src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -445,6 +482,25 @@ options::client_session get_session_options(document::view object) {
     return session_opts;
 }
 
+options::client_encryption get_client_encryption_options(document::view object,
+                                                         client* key_vault_client) {
+    options::client_encryption ce_opts;
+    ce_opts.key_vault_client(key_vault_client);
+    ce_opts.key_vault_namespace({"keyvault", "datakeys"});
+    ce_opts.kms_providers(make_kms_doc());
```

Comment:
`keyVaultClient` and `kmsProviders` are parameters that are specified by the client encryption entity definition. Additionally, TLS opts below should not be set unless the KMIP KMS provider is provided. See [C Driver implementation](https://github.com/mongodb/mongo-c-driver/blob/72cba009ec70fd734dc76426c5385487a1c6930d/src/libmongoc/tests/unified/entity-map.c#L799) for reference.